### PR TITLE
Kyle

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

**Comment by sweetstyle11** [comment]
I guess I could look at the solution but I still want to try and figure this on out on my own. I'm working on Atomic 5, and I'm trying to return the user's input. My first thought would be to loop through their input as they type it but that seems odd. Would you mind pointing me in the right direction/ maybe give me something to google first before I look at the solution?

**Comment by sumeetjain** [comment]
What event (that is, what user action) are you trying to capture? Try putting that into words and then googling for a way to listen for it.

**Comment by halfghaninne** [comment]
And if that gets a bit hard to parse through, W3 has a [list of HTML DOM Events](http://www.w3schools.com/jsref/dom_obj_event.asp)

**Comment by sweetstyle11** [comment]
Thanks that helped. I actually did end up having to look at the solution, the research I found said to use onKeyPress and onKeyUp and it looked like they were using those inside the html tags like this:  <input id="full_name" type="text" onKeyPress="myMethod()" onKeyUp="myMethod()">. This did not work for me. Why would they put javascript inside and html tag? That doesn't seem like good practice.

**Comment by sweetstyle11** [comment]
Nevermind, Wendy helped me figure out why.
